eduNEXT/frontend-app-validation-panel
# fix: filtering properly by categories

## Description
This PR allows an appropriate visualization of the categories in the Data Table, and allows filtering by every category in an independent way.

## How to test
1. Create 2 validation processes, one of them with 2 categories and the other one with only 1 (both should share one category)
![image](https://github.com/eduNEXT/frontend-app-validation-panel/assets/86393372/87dc33d4-4a98-4664-940e-de29e5485c1b)
2. Then go to the categories filter; you'll be able to watch the categories available to filter
![image](https://github.com/eduNEXT/frontend-app-validation-panel/assets/86393372/b356511a-97b1-45dd-a67d-c65c4605a47d)

3. Select the category that the courses are sharing and you will watch both of the validation processes because they are related to the category
![image](https://github.com/eduNEXT/frontend-app-validation-panel/assets/86393372/0a18bf2f-60d6-4b30-ba14-b0ace52bd0c0)

4. Unselect the previous category and select the other one, now you will watch only the course that is related to the category selected
![image](https://github.com/eduNEXT/frontend-app-validation-panel/assets/86393372/0c20cadd-e65a-4727-a891-872a4e9d0bdc)
